packages/vite/src/node/server/index.ts

```diff
@@ -1278,7 +1280,8 @@ async function restartServer(server: ViteDevServer) {
 
   if (shortcutsOptions) {
     shortcutsOptions.print = false
-    bindCLIShortcuts(server, shortcutsOptions)
+    // Skip environment checks since shortcuts were bound before restart
+    bindCLIShortcuts(server, shortcutsOptions, true)
```

Comment:
```suggestion
    bindCLIShortcuts(
      server,
      shortcutsOptions,
      // Skip environment checks since shortcuts were bound before restart
      true,
    )
```
Just to make it clear which parameter this comment is referring to.